patches/chromium/remove_--no-untrusted-code-mitigations_from_content_and_gin.patch

```diff
@@ -0,0 +1,87 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Kenichi Ishibashi <bashi@chromium.org>
+Date: Mon, 16 Aug 2021 03:24:27 +0000
+Subject: Remove --no-untrusted-code-mitigations from //content and //gin
+
+The V8 flag was removed in crrev.com/c/3045704.
+
+Bug: 1003890
+Change-Id: Ic2dc0bfa982609bb71ae3fa0b72b47b7c456cd12
+Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3096585
+Reviewed-by: Matt Falkenhagen <falken@chromium.org>
+Reviewed-by: Kentaro Hara <haraken@chromium.org>
+Commit-Queue: Kenichi Ishibashi <bashi@chromium.org>
+Cr-Commit-Position: refs/heads/master@{#912056}
```

Comment:
Is this a backport?

Reply:
@nornagon looks like this patch was dropped before the merge by this commit:
https://github.com/electron/electron/commit/aaaa51a5ec4c325073dcfdcecc9bf5e2e699cc4b